app/src/main/java/com/jojjnator/kalkylator/di/AppModule.kt

```diff
@@ -2,8 +2,10 @@ package com.jojjnator.kalkylator.di
 
 import org.koin.dsl.module
 import com.jojjnator.kalkylator.viewmodel.CalculatorViewModel
+import com.jojjnator.kalkylator.viewmodel.GoTViewModel
 import org.koin.androidx.viewmodel.dsl.viewModel
 
 val appModule = module {
     viewModel { CalculatorViewModel() }
+    viewModel{ GoTViewModel() }
```

Comment:
Change the name to GameOfThronesViewModel?

Reply:
Sounds smart, changed to QuoteViewModel even